packages/core-macro/src/component_body_deserializers/inline_props.rs

```diff
@@ -102,7 +102,9 @@ fn get_props_struct(component_body: &ComponentBody) -> ItemStruct {
         quote! { #generics }
     };
 
+    let doc = format!("Properties for the [`{fn_ident}`] component.");
```

Comment:
Could you apply the same change to the component macro [here](https://github.com/DioxusLabs/dioxus/blob/a0b95a82a0fa16d9101897554444878da7096f74/packages/core-macro/src/component_body_deserializers/component.rs#L63C20-L63C20)?

Reply:
In that case, in order to avoid having the doc comment appearing twice, I would have to revert the change in the `inline_props.rs` file, since `component.rs` is indirectly referencing the `get_props_struct()` function via the `InlinePropsDeserializerOutput` type. But I guess that is fine if that macro is deprecated anyway!